packages/esm-commons-lib/src/utils/encounter-list-config-builder.ts

```diff
@@ -75,7 +76,26 @@ export const getTabColumns = (columnsDefinition: Array<ColumnDefinition>) => {
     key: column.id,
     header: column.title,
     concept: column.concept,
-    getValue: (encounter) => {
+    getValue: async (encounter) => {
+      if (column.concept) {
+        switch (column.concept) {
+          case 'treatmentId':
+            return await getTbTreatmentId(encounter, 'treatmentId');
```

Comment:
We don't want to have specific names like getTB in commons lib, we should make a generic function that handle these cases.